hack/gather-registry-artifacts.sh

```diff
@@ -0,0 +1,13 @@
+#! /bin/bash
+
+set -eou pipefail
+set -x
+
+CATALOG_SOURCE_NAMESPACE=${CATALOG_SOURCE_NAMESPACE:=openshift-marketplace}
+CATALOG_SOURCE_NAME=${CATALOG_SOURCE_NAME:=redhat-operators}
+
+OUTPUT_DIRECTORY=${OUTPUT_DIRECTORY:=$(mktemp -d)/resources}
```

Comment:
```suggestion
OUTPUT_DIRECTORY=${OUTPUT_DIRECTORY:=$(mktemp -ud)/resources}
```
`-u` is dry run, avoids creating a directory twice